source/src/index.html

```diff
@@ -536,6 +539,7 @@ <h4 id="enable-disable-alarmsound" class="text" part="enable-disable-alarmsound"
             right:0;
             cursor: pointer;
             opacity: 0.33;
+            transition: transform .3s ease;
```

Comment:
Is this already included in the html fragments?

Reply:
I only changed the css in index.html, just updated the html fragments!